core/checks.py

```diff
@@ -796,6 +798,21 @@ def _get_running_services(self):
                         self.services[svc]["ps_cmds"].append(ret.group(0))
                         break
 
+    def _get_masked_services(self):
+        """Get masked services."""
+        masked = re.compile('^([^\\s]+)\\s*masked')
+        core_services = [
+            'nova',
+            'neutron'
```

Comment:
Any reason not to use https://github.com/dosaboy/hotsos/blob/ac566dc176ba5fb2a1098b4a21e01f2c555bacc6/core/plugins/openstack/__init__.py#L186?

That way we can easily do this for any service without having to define statically here.